zmon-agent.py

```diff
@@ -158,8 +157,8 @@ def get_running_apps(region):
                 ins['runtime'] = user_data['runtime']
 
                 if 'StackVersion' in tags:
-                    ins['stack'] = tags['Name']
-                    ins['resource_id'] = tags['aws:cloudformation:logical-id']
+                    ins['stack'] = tags.get('Name', None)
```

Comment:
the "None" is redundant in both cases: just use tags.get('Name') (it will return None by default if item not found)